Added Revit 2021 build targets & assembly references for dotnet48

GertjanVDBVK opened the pull request:
All i did was add some additional stuff to the projects in the solution. I hope this allows people to build their own 2021 revit python shell.
I know you won't be maintaining this anymore, but I was hoping we could still get a working 2021 build. As a sendoff.

Wish I could do more, but my current knowledge on the matter is a bit too sparse I fear.

daren-thomas reviewed (changes requested):
@GertjanVDBVK this is brilliant! Due to missing Revit licenses, I can't actually test it, but it seems to be alright! Would you mind updating the installer too? This file to edit would be https://github.com/architecture-building-systems/revitpythonshell/blob/master/Setup_RevitPythonShell_2020.iss - copy it and replace the 2020 stuff with 2021? This could really help a lot of people! :-) great job!!

GertjanVDBVK commented:
@daren-thomas I gave it a try. I hope it's correct. I'm very new to git and I'm kind of afraid to break things.
Anyway, I've added the .iss file and replaced every single 2020 reference to 2021

daren-thomas commented:
Wow thank you! Did you build the executable? (I'm afraid I can't anymore since I don't have that Revit version...)

> I don't even know how you managed that voodoo magic

Pretty sure that wasn't me... ;-)

I'll merge this now. Thank you very very much!
